script.js

```diff
@@ -1,4 +1,116 @@
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// initialise game mode
+var currentGameMode = "choose order of dice";
+var user = "Player 1";
+var diceNumberPlayer1 = 0;
+var diceNumberPlayer2 = 0;
+var diceRoll1 = 0;
+var diceRoll2 = 0;
+
+var whoWins = function (diceNumberPlayer1, diceNumberPlayer2) {
+  if (diceNumberPlayer1 > diceNumberPlayer2) {
+    return "Player 1 wins. Click submit to play again";
+  } else if (diceNumberPlayer2 > diceNumberPlayer1) {
+    return "Player 2 wins. Click submit to play again";
+  } else if (diceNumberPlayer1 == diceNumberPlayer2) {
+    return "It is a draw. Click submit to play again";
+  }
+};
+
+var diceRollSame = function (user, diceRoll1, diceRoll2) {
+  return `Welcome ${user}. <br> You rolled ${diceRoll1} for Dice 1 and ${diceRoll2} for Dice 2. <br> Your number is ${diceRoll1}${diceRoll2}.`;
+};
+
+var diceRollDiff = function (user, diceRoll1, diceRoll2) {
+  return `Welcome ${user}. <br> You rolled ${diceRoll1} for Dice 1 and ${diceRoll2} for Dice 2. Choose the order of the dice.`;
+};
+
+var dice1First = function (user, diceRoll1, diceRoll2) {
+  output = `You chose Dice 1 first. Your number is ${diceRoll1}${diceRoll2}.`;
+  if (user == "Player 1") {
+    output = output + "<br> It is now Player 2's turn.";
+    return output;
+  }
+};
+
+var dice2First = function (user, diceRoll1, diceRoll2) {
+  output = `You chose Dice 2 first. Your number is ${diceRoll2}${diceRoll1}.`;
+  if (user == "Player 1") {
+    output = output + "<br> It is now Player 2's turn.";
+    return output;
+  }
+};
```

Comment:
potentially this can be made into one function taking the input as the user choice and outputing a varied message based on the output